pkgs/os-specific/linux/cpupower/gui.nix

```diff
@@ -0,0 +1,95 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, desktop-file-utils
+, gettext
+, glib
+, gobject-introspection
+, gtk3
+, hicolor-icon-theme
+, meson
+, ninja
+, pkg-config
+, python3
+, systemd
+, wrapGAppsHook
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "cpupower-gui";
+  version = "0.8.0";
+
+  # This packages doesn't have a setup.py
+  format = "other";
+
+  src = fetchFromGitHub {
+    owner = "vagnum08";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1rrzlcmspill7qivv3bg8v810ppj4wxa9jcf2ddrjqcbq4lfxayk";
+  };
+
+  nativeBuildInputs = [
+    desktop-file-utils # needed for update-desktop-database
+    gettext
+    glib # needed for glib-compile-schemas
+    gobject-introspection # need for gtk namespace to be available
+    hicolor-icon-theme # needed for postinstall script
+    meson
+    ninja
+    pkg-config
+    python3
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    glib
+    gtk3
+  ];
+
+  propagatedBuildInputs = with python3.pkgs; [
+    dbus-python
+    pygobject3
+  ];
+
+  mesonFlags = [
+    "-Dsystemddir=${placeholder "out"}/lib/systemd"
+  ];
+
+  preConfigure = ''
+    patchShebangs build-aux/meson/postinstall.py
+  '';
+
+  strictDeps = false;
+  dontWrapGApps = true;
+
+  # Polkit rules aren't installed and we need them to authorize users via polkit
+  # we refer to this file in the corresponding nixos module
+  # See also https://github.com/vagnum08/cpupower-gui/issues/28
+  postInstall = ''
+    mkdir ${placeholder "out"}/share/polkit-1/rules.d/
+    cp ../data/polkit/cpupower-gui.rules ${placeholder "out"}/share/polkit-1/rules.d/
+  '';
+
+  preFixup = ''
+    makeWrapperArgs+=("''${gappsWrapperArgs[@]}")
+  '';
+
+  postFixup = ''
+    wrapPythonProgramsIn $out/lib "$out $propagatedBuildInputs"
+
+    # NOTE The following may not be necessary in version 0.9.0+
+    # Remove unused pkla file
+    rm ${placeholder "out"}/var/lib/polkit-1/localauthority/10-vendor.d/org.rnd2.cpupower-gui.pkla
+    # And empty directories
+    rmdir -p --ignore-fail-on-non-empty ${placeholder "out"}/var/lib/polkit-1/localauthority/10-vendor.d
+  '';
```

Comment:
throughout here you used `placeholder out` in the phases, and this really isn't needed in the phases and you can safely use `$out` in the particular expression